pkg/arvo/app/btc-node-store.hoon

```diff
@@ -0,0 +1,146 @@
+::  btc-node-store: data store for state received from a bitcoin full node
+::
+/-  *btc-node-store
+/+  *btc-node-json, default-agent, verb
+::
+=>  |%
+    ::
+    +$  card  card:agent:gall
+    ::
+    +$  state
+      $%  [%0 state-zero]
+      ==
+    ::
+    +$  state-zero
+      $:  =wallets
+          default-wallet=@t
+      ==
+    --
+::
+=|  state-zero
+=*  state  -
+::  Main
+::
+%+  verb  |
+^-  agent:gall
+=<  |_  =bowl:gall
+    +*  this      .
+        btc-core  +>
+        bc        ~(. btc-core bowl)
+        def       ~(. (default-agent this %|) bowl)
+    ::
+    ++  on-init
+      ^-  (quip card _this)
+      :-  ~
+      %=    this
+        wallets  (~(put by wallets) [*@t *wallet])
+      ==
+    ::
+    ++  on-save   !>(state)
+    ++  on-load
+      |=  old=vase
+      `this(state !<(state-zero old))
+    ::
+    ++  on-poke
+      |=  [=mark =vase]
+      ^-  (quip card _this)
+      ~&  got-poked-with-data=mark
+      |^
+      ?+    mark    (on-poke:def mark vase)
+          %btc-node-store-action
+        (store-action !<(btc-node-store-action vase))
+      ::
+          %btc-node-store-command
+        (store-command !<(btc-node-store-command vase))
+      ==
+      ::
+      ++  store-action
+        |=  action=btc-node-store-action
+        ^-  (quip card _this)
+        =^  cards  state
+          ?+  -.action  ~|  [%unsupported-action -.action]  !!
```

Comment:
```suggestion
          ?+  -.action  ~|([%unsupported-action -.action] !!)
```

Slightly more immediately obvious that it's just one thing (instead of something that flows over to the next line).